apps/site/assets/ts/schedule/__tests__/MapTest.tsx

```diff
@@ -77,7 +79,7 @@ describe("reducer", () => {
 
   it("resets markers", () => {
     const result = reducer(
-      { markers: data.markers, shapeId: "1", channel: "vehicle:1:1" },
+      { markers: data.markers, shapeIds: ["1"], channel: "vehicle:1:1" },
```

Comment:
Small thing: this shapeIds key appears to come last in the object in every other test in this file, making this one inconsistent in appearance.